QRCoder/SvgQRCode.cs

```diff
@@ -108,96 +117,85 @@ public string GetGraphic(Size viewBox, string darkColorHex, string lightColorHex
     {
         int offset = drawQuietZones ? 0 : 4;
         int drawableModulesCount = QrCodeData.ModuleMatrix.Count - (drawQuietZones ? 0 : offset * 2);
-        double pixelsPerModule = Math.Min(viewBox.Width, viewBox.Height) / (double)drawableModulesCount;
-        double qrSize = drawableModulesCount * pixelsPerModule;
-        string svgSizeAttributes = (sizingMode == SizingMode.WidthHeightAttribute) ? $@"width=""{viewBox.Width}"" height=""{viewBox.Height}""" : $@"viewBox=""0 0 {viewBox.Width} {viewBox.Height}""";
-        ImageAttributes? logoAttr = null;
-        if (logo != null)
-            logoAttr = GetLogoAttributes(logo, viewBox);
 
-        // Merge horizontal rectangles
-        int[,] matrix = new int[drawableModulesCount, drawableModulesCount];
-        for (int yi = 0; yi < drawableModulesCount; yi += 1)
+        // Build SVG opening tag with size attributes
+        var svgFile = new StringBuilder();
+
+        svgFile.Append(@"<svg xmlns=""http://www.w3.org/2000/svg"" viewBox=""0 0 ");
+        Append(drawableModulesCount);
+        svgFile.Append(' ');
+        Append(drawableModulesCount);
+        svgFile.Append(@""" shape-rendering=""crispEdges""");
```

Comment:
IMO this reads better. There are too many `"` for me 😉.
```suggestion
        svgFile.Append(@"\" shape-rendering=\"crispEdges\"");
```
(or using raw string literals).

I know that there are other places where it's fine as written, but here it starts and ends with 3 `"`, so my brain switched a bit into raw string literal mode, which is not correct.
W/ the suggestion it's clearer and easier to follow in this place.

Reply:
Changed all to use regular strings.  Raw strings are not possible when a string ends with a quote (unless using the multi-line variation).